src/ecs/resources/input.rs

```diff
@@ -28,7 +41,7 @@ impl<'a> Iterator for PressedKeys<'a> {
 
 /// An iterator over the currently pressed down mouse buttons.
 pub struct PressedMouseButtons<'a> {
-    iterator: Keys<'a, MouseButton, KeyQueryState>,
+    iterator: Iter<'a, MouseButton>,
```

Comment:
This conceals the implementation details a bit better, but other than that no not really.  When I opened the file it was a wrapper struct so I left it that way.  I'm happy to change it either way.